extensions/amp-facebook/1.0/storybook/Basic.js

```diff
@@ -9,8 +9,8 @@ import {
 } from '@storybook/addon-knobs';
 
 export default {
-  title: 'Facebook',
-  component: Facebook,
+  title: 'BentoFacebook',
```

Comment:
I think this can stay `Facebook` so that the Storybook pages wouldn't all start with `Bento`